### PR TITLE
[WIP] convert services.yaml to bytes before dumping to yaml

### PR DESCRIPTION
**Pull request by mjksmith**
The fact that our tests didn't catch this is bad. I want to add more tests to this before shipping & unpinning.

If you absolutely need to unpin you can merge this (I tested it locally) and watch services.yaml for any weirdness as it deploys.

### CODE REVIEW

**Comment by chriskuehl** [comment]
I added a test in #925.

File: ..-UTcARg [hunk 1]
```diff
@@ -0,0 +1,1255 @@
+---
```
Comment (chriskuehl) [inline ..-UTcARg]: i guess you didn't mean to commit this?

Comment (mjksmith) [inline ..-UTcARg]: 👍

**Comment by solarkennedy** [comment]
@mjksmith just merge in #925 into this branch and see the tests pass.

**Pull request merged**